orchagent/vnetorch.cpp

```diff
@@ -1045,6 +1053,7 @@ bool VNetRouteOrch::doRouteTask<VNetVrfObject>(const string& vnet, IpPrefix& ipP
         {
             syncd_tunnel_routes_.erase(vnet);
         }
+        removePriorityEndpoints(vnet, ipPrefix);
```

Comment:
You are invoking this after erasing `ipPrefix `from `syncd_tunnel_routes_ `but this is being accessed in your function. The could become an invalid access. Please move this to line 1050.

Reply:
that correct. Ill fix that.